test/integration/auth.spec.ts

```diff
@@ -1154,6 +1154,55 @@ describe('admin.auth', () => {
     });
   });
 
+  describe('Project config management operations', () => {
+    before(function() {
+      if (authEmulatorHost) {
+        this.skip(); // getConfig is not supported in Auth Emulator
+      }
+    });
+    const projectOption1: UpdateProjectConfigRequest = {
```

Comment:
Nit: would `projectConfig1` be a better name for this?

Reply:
Updated